commands/ReviewCommand.js

```
@@ -283,7 +283,7 @@ export class ReviewCommand extends Commander {
       verdict: finalLabel,
       items: duplicateResults.items,
       cleanups: cleanupResults.cleanups || [],
-      message: summary.join('\n'),
+      display: summary.join('\n'),
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)